pkg/resource/resource.go

```diff
@@ -35,6 +35,7 @@ const (
 	MirrorTopic                 = "mirror topic"
 	Network                     = "network"
 	Organization                = "organization"
+	Peering                     = "peering"
```

Comment:
Do we want to call this a "peering connection"?

Reply:
Looks like our api documentation uses just "peering", should I change all of the "peering connection" to "peering" if thats the case, to keep things consistent?

Reply:
yes please!

Reply:
Thank you! Removed the word `connection` and updated the relevant golden files as well.